version.json

```diff
@@ -1,3 +1,3 @@
 {
-    "version": "0.48.1"
+    "version": "0.48.2"
```

Comment:
For future refactoring but this should be auto generated using the same version as package.json